bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -126,7 +126,15 @@ def RCanvasAvailable():
        return False
    return True
 
-_enableJSVis = True
+def initializeJSVis():
+    global _enableJSVis
+    jsroot_default = ROOT.gEnv.GetValue("JSROOT.Enable", "on").lower()
```

Comment:
I would use "Jupyter.JSRoot" as name of such variable - to avoid confusion with other ROOT components